src/map/script.cpp

```diff
@@ -11490,9 +11490,8 @@ BUILDIN_FUNC(killmonster)
 		}
 	}
 
-	map_freeblock_lock();
+	FreeBlockLock freeLock;
```

Comment:
```suggestion
```
map_foreachinmap already has a FreeBlockLock included.
This can safely be removed.